xtos/include/sof/lib/perf_cnt.h

```diff
@@ -55,8 +55,11 @@ struct perf_cnt_data {
  * "cpu" timestamps to be equal.
  */
 #ifdef __ZEPHYR__
-	#ifdef CONFIG_TIMING_FUNCTIONS
-		#define perf_cnt_get_cpu_ts arch_timing_counter_get
+
+#include <zephyr/timing/timing.h>
+
+#ifdef CONFIG_TIMING_FUNCTIONS
```

Comment:
indentation... would be better to either leave as is or fix consistently

Reply:
ack, @lyakh can you recheck?